### ISSUE TITLE
read *.fq.gz or *.fq.bz2

### ISSUE THREAD

**Issue by wcxiaoping** [jkbonfield/fqzcomp#3]
Can this software read FASTQ files compressed with gzip or bzip2 directly?

**Comment by jkbonfield** [comment]
No, but it can read from stdin so you can do `zcat file.gz | fqz_comp`.
